src/app.ts

```diff
@@ -78,7 +81,7 @@ export function generateServerlessRouter(
         mainRouter.use(cors(corsOptions));
         hasCORSEnabled = true;
     }
-
+    const mainRoute = createMainRoute(mainRouter, multiTenancyOptions);
```

Comment:
`mainRoute` vs `mainRouter` in this file confusing, especially when some `mainRoute` were renamed to `mainRouter` while others were not.